[2023-12-27 18:22:25,747] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: OpenWeather_ETL2.extraer_data manual__2023-12-27T18:22:21.691707+00:00 [queued]>
[2023-12-27 18:22:25,768] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: OpenWeather_ETL2.extraer_data manual__2023-12-27T18:22:21.691707+00:00 [queued]>
[2023-12-27 18:22:25,769] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-27 18:22:25,770] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-12-27 18:22:25,771] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-27 18:22:25,807] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): extraer_data> on 2023-12-27 18:22:21.691707+00:00
[2023-12-27 18:22:25,816] {standard_task_runner.py:52} INFO - Started process 14825 to run task
[2023-12-27 18:22:25,825] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'OpenWeather_ETL2', 'extraer_data', 'manual__2023-12-27T18:22:21.691707+00:00', '--job-id', '129', '--raw', '--subdir', 'DAGS_FOLDER/ETL_OpenWeather2.py', '--cfg-path', '/tmp/tmpn3z0pd7q', '--error-file', '/tmp/tmp8lsylu2a']
[2023-12-27 18:22:25,828] {standard_task_runner.py:80} INFO - Job 129: Subtask extraer_data
[2023-12-27 18:22:26,011] {task_command.py:371} INFO - Running <TaskInstance: OpenWeather_ETL2.extraer_data manual__2023-12-27T18:22:21.691707+00:00 [running]> on host 44a36bef83ad
[2023-12-27 18:22:26,163] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2023-12-27 18:22:26,283] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Furtado_Julio
AIRFLOW_CTX_DAG_ID=OpenWeather_ETL2
AIRFLOW_CTX_TASK_ID=extraer_data
AIRFLOW_CTX_EXECUTION_DATE=2023-12-27T18:22:21.691707+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-12-27T18:22:21.691707+00:00
[2023-12-27 18:22:26,285] {logging_mixin.py:115} INFO - Adquiriendo data para la fecha: 2023-12-27 18
[2023-12-27 18:22:27,416] {logging_mixin.py:115} INFO - Datos adquiridos!
[2023-12-27 18:22:27,428] {python.py:173} INFO - Done. Returned value was: None
[2023-12-27 18:22:27,470] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=OpenWeather_ETL2, task_id=extraer_data, execution_date=20231227T182221, start_date=20231227T182225, end_date=20231227T182227
[2023-12-27 18:22:27,525] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-27 18:22:27,677] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
